pkg/kubectl/cmd/cmd.go

```diff
@@ -235,6 +236,7 @@ Find more information at https://github.com/kubernetes/kubernetes.`,
 	cmds.AddCommand(NewCmdVersion(f, out))
 	cmds.AddCommand(NewCmdExplain(f, out))
 	cmds.AddCommand(NewCmdConvert(f, out))
+	cmds.SetOutput(os.Stdout)
```

Comment:
@kargakis : If you see the example command line to recreate the problem, with and without this fix, you will see why we need this additional line of code:
https://github.com/kubernetes/kubernetes/issues/13522#issue-104549046

Also, tracing through the code, you will see that unless we call SetOutput, we'll end up here:
https://github.com/kubernetes/kubernetes/blob/master/vendor/github.com/spf13/cobra/command.go#L143